### PR TITLE
fix(api-client): view layout z index

### PR DESCRIPTION
**Pull request by antlio**
this pr fixes the variable value tooltip visibility when used within the address bar as seen below:

**⊢ before / after**
<div>
image
image
</div>

### CODE REVIEW

File: packages/api-client/src/components/ViewLayout/ViewLayout.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="flex flex-col min-h-0 flex-1 *:border-t-1/2 first:*:border-t-0 md:*:border-t-0 xl:overflow-hidden md:flex-row leading-3">
+    class="flex flex-col min-h-0 flex-1 *:border-t-1/2 first:*:border-t-0 md:*:border-t-0 xl:overflow-hidden md:flex-row leading-3 z-0">
```
[the remark this answers is not in the document]
Comment (antlio) [reply packages/api-client/src/components/ViewLayout/ViewLayout.vue:3]: seems like it does but not against another take here if there's one!

[the remark this answers is not in the document]
Comment (hwkr) [reply packages/api-client/src/components/ViewLayout/ViewLayout.vue:3]: Ahh I'm guessing it's this

Google Chrome-2024-11-19-12-04-58@2x

**Comment by hwkr** [comment]
Is is possible to raise the z-index of the tooltips instead? The element that's clipping the tooltip only has a z-index of 1 as far as I can tell and tooltip should probably be at least z-context if not z-overlay

File: packages/api-client/src/components/ViewLayout/ViewLayout.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (antlio) [reply packages/api-client/src/components/ViewLayout/ViewLayout.vue:3]: i think that this might has been introduced by #3871

**Comment by amritk** [comment]
>Is is possible to raise the z-index of the tooltips instead? The element that's clipping the tooltip only has a z-index of 1 as far as I can tell and tooltip should probably be at least z-context if not z-overlay

z-tooltip? should be pretty high

File: packages/components/src/components/ScalarTooltip/ScalarTooltip.vue [hunk 2]
```diff
@@ -50,7 +50,7 @@ defineEmits<{
         <TooltipContent
           v-if="!props.disabled"
           :align="props.align"
-          class="scalar-app"
+          class="scalar-app z-context"
```
Comment (hwkr) [inline packages/components/src/components/ScalarTooltip/ScalarTooltip.vue:53]: 🚀

**Review by hwkr — APPROVED** [review]
Looks good!

**Pull request merged**